packages/framework/src/Component/Image/View/ImageView.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Image\View;
+
+/**
+ * @experimental
+ *
+ * Class representing images in FE templates (to avoid usage of Doctrine entities a hence achieve performance gain)
+ */
+class ImageView
```

Comment:
We discussed this back and forth, however, in the end, we decided not to implement it this way because of the need for data pre-generation for all sizes.